### ISSUE TITLE
Add ability to generate QR code of username and password fields

### ISSUE THREAD

**Issue by matthiasduyck** [Kunzisoft/KeePassDX#729]
When trying to quickly authenticate on another device that is either not setup with your database and or KeePass DB reading application because it is new or not your own, it can be quite a hassle. Typing over credentials or installing the application and getting the DB can be annoying when you have to be quick. Most operating systems do have built in or easy ways to scan a QR code however, which could solve this problem.

By adding a 'generate QR' button next to the existing copy value button, a simple plaintext QR code is generated that contains that fields' value. This can then be used to scan on the other device and does not require any software or files(such as your own DB) to be available. Both Android and iOS have this functionality built-in. On Windows, MacOS and others, you might still need to install a scanning application, but this still alleviates the need to share the DB file.

This experience could be further improved/expanded upon by software on the receiving/scanning end of this solution. But the key is that it is not required.

Similar functionality is provided by some plugins for the desktop app such as: https://github.com/JanisEst/KeePassQRCodeView

**Comment by Ilithy** [comment]
I sign up for this feature, very handy to easily share a username and/or password;
 if possible, with the possibility to store the QR code in the entry (with the option to always show it, or hide it, as for the custom fields).

**Comment by hokonch** [comment]
You can select a password or username and share it with the QR Code Reader app to generate a QR code. I think this is basically a good solution, but the password field can only be selected on the edit screen.
It would be more convenient to be able to select protected fields when trusting the clipboard.

> the possibility to store the QR code in the entry 

Related to #473

**Comment by Ilithy** [comment]
@hokonch: Yes, if it allows you to add a custom field that allows you to view the QR code directly in the entry. (To be more precise)

**Comment by J-Jamet** [comment]
> Exactly, the problem right now is that you have to trust the application generating the QR code, and trust the application to display it, (for the moment KeepassDX is only used to store the file securely), it's annoying.

If this is a trust issue, using a library in the app also has this problem. So it's only moving the issue and forcing the user to use a single QR library. In the case of an external QR code application, the user has the choice and can use an open source one.
